### DISCUSSION TITLE
hELLO  건의사항

### DISCUSSION THREAD

**Discussion by E-know** [pronist/hELLO#88] [Ideas]
안녕하세요 개발자님. 만들어주신 티스토리 테마 아주 잘 쓰고 있습니다.

**건의 사항**
(다크모드 사용시)링크를 걸어두면 보통 파랗게 표시가되어서 직관적인데 

링크를 걸어두어도 링크가 파랗게 표시가 안되어서 직관적이지 않다고 생각됩니다.

a[href] 태그 이용해서 바꿀려해도 잘 안되어서 건의사항에 남겨둡니다. 

언제나 행복하십쇼!

**Comment by pronist** [comment]
생각해보니 그렇군요. 
다음 패치 때  이 사항은 반영해보도록 하겠습니다. 다만 파란색이 아니라 다른 색일 수도 있습니당 :)

의견 남겨주셔서 고맙습니다.

**Comment by pronist** [comment]
본문에 한정하여 링크가 파란색으로 표시되도록 변경되었습니다. :)
